cli/commandhandler.h

```diff
@@ -64,49 +64,54 @@ bool init()
 
 void handleLoadPemFile(const std::map<uint32_t, std::string> &userInputs)
 {
-    ih::wrapper::PemHandlerInputWrapper const pemInputWrapper(userInputs);
-    ih::PemFileReader pemHandler(pemInputWrapper.getPemFilePath());
-
-    if (pemHandler.isFileValid())
+    try
     {
-        std::cerr << "Bech32 address: " << pemHandler.getAddress().getBech32Address() << "\n";
+        ih::wrapper::PemHandlerInputWrapper const pemInputWrapper(userInputs);
+        ih::PemFileReader pemHandler(pemInputWrapper.getPemFilePath());
+        std::cerr << "File loaded successfully! Bech32 address: " << pemHandler.getAddress().getBech32Address() << "\n";
     }
-    else
+    catch (std::exception const &exception)
     {
-        reportError(ERROR_PEM_INPUT_FILE);
+        std::cerr<< exception.what();
     }
 }
 
 void handleCreateSignedTransactionWithPemFile(const std::map<uint32_t, std::string> &userInputs)
 {
-    ih::wrapper::PemHandlerInputWrapper const pemInputWrapper(userInputs);
     ih::wrapper::TransactionInputWrapper const transactionInputWrapper(userInputs);
-
     ih::JsonFileHandler jsonHandler(transactionInputWrapper.getOutputFile());
-    ih::PemFileReader pemFileHandler(pemInputWrapper.getPemFilePath());
 
-    if (!pemFileHandler.isFileValid())
-    {
-        reportError(ERROR_PEM_INPUT_FILE);
-    }
-    else if (!jsonHandler.isFileValid())
+    if (!jsonHandler.isFileValid())
     {
         reportError(ERROR_JSON_OUT_FILE);
     }
     else
     {
-        Transaction transaction
-                (transactionInputWrapper.getNonce(), transactionInputWrapper.getValue(), transactionInputWrapper.getReceiver(),
-                 pemFileHandler.getAddress(), transactionInputWrapper.getGasPrice(), transactionInputWrapper.getGasLimit(),
-                 transactionInputWrapper.getData(), transactionInputWrapper.getChainId(), transactionInputWrapper.getVersion());
-
-        Signer signer(pemFileHandler.getPrivateKey());
-        transaction.applySignature(signer);
-        jsonHandler.writeDataToFile(transaction.getSerializedTransaction());
+        try
```

Comment:
Moved` try-catch `from here to main.
However, should we really catch every particular error in CLI? Because every error that could happen is because of a user's wrong input. We should not correct them, only report them, right? The user is the one that should take care of handling its own errors.